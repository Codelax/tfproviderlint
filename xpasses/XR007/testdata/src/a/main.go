package a

import (
	"os/exec"
)

func f() {
	failing := exec.Command // want "avoid os/exec.Command"
	// Comment ignored

	//lintignore:XR007
	exec.Command("true")

	exec.Command("true") //lintignore:XR007

	// Failing

	exec.Command("true") // want "avoid os/exec.Command"

	failing("true")
}
