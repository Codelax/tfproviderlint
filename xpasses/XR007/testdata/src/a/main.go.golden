package a

import (
	"os/exec"
)

func f() {
	failing := // want "avoid os/exec.Command"
		// Comment ignored

		//lintignore:XR007
		exec.Command("true")

	exec.Command("true") //lintignore:XR007

	// Failing

	// want "avoid os/exec.Command"

	failing("true")
}
